ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -68,13 +67,13 @@ openMempool registry chainDB cfg = do
 -- | Internal state in the mempool
 data InternalState blk = IS {
       -- | Transactions currently in the mempool
-      isTxs :: TxSeq (GenTxId blk, GenTx blk)
+      isTxs   :: TxSeq (GenTxId blk) (GenTx blk)
```

Comment:
I'm afraid I don't understand the purpose of this patch at all. What does it buy us? It seems to make `TxSeq` more specific with no gain.

Reply:
I'd agree that it doesn't really buy anything. I think I was mostly concerned with treating a `(GenTxId, blk, GenTx blk)` as what would be considered a `tx` from the `TxSeq` point of view. I've dropped this patch from the PR.